### PR TITLE
[WEB-8746] Users in ad-hoc report have overlapping items

### PR DESCRIPTION
**Pull request by vomajda**
http://tribehr1.se4.eng.netsuite.com:8080/browse/WEB-8746
- removed default width (if not specified in DB) from view and added default width for 'text' column into controller
- applying column width to the columns (not just using it to compute table width)
- CSS cleanup - removed duplicities, nowrap by default, ellipsis overflow
- added jquery hover to display hidden overflown content in table cells
- fixed label overflow for long labels and short values
- code cleanup - removed duplicate and unused code, replaced "endfors" and "endifs" by {}
- sorry for the whitespaces :-/

### CODE REVIEW

File: View/AdHocReports/report_display.ctp [hunk 1]
```diff
@@ -2,10 +2,11 @@
 /**
  * Copyright (c) 2013 TribeHR Corp - http://tribehr.com
  * Copyright (c) 2012 Luis E. S. Dias - www.smartbyte.com.br
- * 
+ *
  * Licensed under The MIT License. See LICENSE file for details.
  * Redistributions of files must retain the above copyright notice.
  */
+echo $this->Html->script(array('https://ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js'));
```
[the remark this answers is not in the document]
Comment (vomajda) [reply View/AdHocReports/report_display.ctp]: Good question! I just copied this line from `wizard.ctp` to be sure I'm not using anything new so in this context I guess it's acceptable.

File: View/AdHocReports/report_display.ctp [hunk 2]
```diff
@@ -19,53 +20,60 @@
...
-                $displayField = ucfirst($displayField);
-                echo $displayField; 
-                ?>
+            <?php foreach ($fieldList as $field) {
+				$displayField = substr($field, strpos($field, '.')+1);
+				$displayField = str_replace('_', ' ', $displayField);
+				$displayField = ucfirst($displayField);
+
+				// minimum column width is either 50 or (length * 9) to prevent label overflow
+                $minWidth = 50;
```
[the remark this answers is not in the document]
Comment (vomajda) [reply View/AdHocReports/report_display.ctp]: You're absolutely right. Removing the variable anyway (because of the comment below).

File: View/AdHocReports/report_display.ctp [hunk 3]
```diff
@@ -19,53 +20,60 @@
...
+				$columnWidth = max(array($tableColumnWidth[$field], $minWidth, strlen($displayField) * 9));
+				// force ID cols to be 50 despite they're not configured like that in DB
+                $columnWidth = strtolower($displayField) == 'id' ?  $minWidth : $columnWidth;
```
[the remark this answers is not in the document]
Comment (vomajda) [reply View/AdHocReports/report_display.ctp]: Removed the condition and replaced spaces by tabs

File: View/Elements/fields_dnd.ctp [hunk 4]
```diff
@@ -1,42 +1,42 @@
...
-							'checked'=>$modelFieldAdd, 
+							'checked'=>$modelFieldAdd,
 							'class'=>'fieldCheckbox',
 							'data-fieldName' => $class . "." . $field
 						)
 					);
 //			echo '</div>';
-			
+
 //			echo '<div>';
```
[the remark this answers is not in the document]
Comment (vomajda) [reply View/Elements/fields_dnd.ctp]: They're gone :)

**Comment by vomajda** [comment]
Closing this PR as it will be superseded by another one.
